ethereum/ethereum-erigon/babel.yaml

```diff
@@ -0,0 +1,111 @@
+version: 0.0.1
+container_uri: docker-daemon://ethereum-erigon:latest
+sku_code: ETH-ERG
+org_id: null
+description: Ethereum Erigon node 
+visibility: public
+protocol_key: ethereum
+variants:
+  # The same protocol can have different implementations. This field let identify what implementation(s) has been used.
+  # Variant name may additional contain protocol network name e.g. "eth-mainnet".
+  # This value will be available in a Rhai runtime via `node_env().variant_key`.
+  - key: erigon-mainnet-archive
+    archive_pointers:
+      - pointer: !store_id ethereum-erigon-mainnet-archive-v1
+      - pointer: !store_id ethereum-erigon-mainnet-archive-v1
+        new_archive_properties: [ "tracing-api-on-value" ]
+    min_cpu: 8
+    min_memory_mb: 32000
+    min_disk_gb: 5000
+  - key: erigon-sepolia-archive
+    archive_pointers:
+      - pointer: !store_id ethereum-erigon-sepolia-archive-v1
+      - pointer: !store_id ethereum-erigon-sepolia-archive-v1 
+        new_archive_properties: [ "tracing-api-on-value" ]
+    min_cpu: 4
+    min_memory_mb: 16000
+    min_disk_gb: 2000
+  - key: erigon-goerli-archive
+    archive_pointers:
+      - pointer: !store_id ethereum-erigon-goerli-archive-v1
+      - pointer: !store_id ethereum-erigon-goerli-archive-v1 
```

Comment:
It doesn't make sense to define `new_archive: true` property if it have the same store_id.

Reply:
What I wanted to achieve here is that whenever you select the tracing API, the dataset should point to the archive store rather than the full one. Can we configure this behavior more explicitly?